data/transactions/logic/eval.go

```diff
@@ -1946,6 +1949,11 @@ func (cx *EvalContext) txnFieldToStack(txn *transactions.Transaction, fs txnFiel
 		sv.Uint = txn.Fee.Raw
 	case FirstValid:
 		sv.Uint = uint64(txn.FirstValid)
+	case FirstValidTime:
+		if cx.EvalParams.FirstValidTimestamp == 0 {
+			return sv, fmt.Errorf("unable to obtain timestamp for %d", txn.FirstValid-1)
```

Comment:
nit: 
- `unable to obtain timestamp for %d` -> `unable to obtain timestamp for round %d`
- instead of `txn.FirstValid`, use txn.FirstValid.Subsaturate(1)

Reply:
done